rh/schema.py

```diff
@@ -37,6 +76,20 @@ class RhForm(DjangoSessionFormMutation):
     class Meta:
         source = RhFormInfo
 
+    @classmethod
+    def perform_mutate(cls, form, info: ResolveInfo):
+        request = info.context
+        result = super().perform_mutate(form, info)
+        form_data = RhFormInfo.get_dict_from_request(request)
+        assert form_data is not None
+
+        full_address = form_data["address"] + ", " + form_data["borough"]
+        bbl, _, _ = lookup_bbl_and_bin_and_full_address(full_address)
+        if bbl:
+            rent_stab_info = process_rent_stab_data(run_rent_stab_sql_query(bbl))
+            request.session[RENT_STAB_INFO_SESSION_KEY] = rent_stab_info
+        return result
```

Comment:
Hmm, shouldn't there be an `else` clause here that sets the rent stab info to an empty dictionary?  Otherwise if the user already has submitted the form with geosearch working, but then goes back and changes their address, but this time geosearch _doesn't_ work, the stale rent stab info for their previously-entered address will still be in the session.

**Update:** rather than an `else` clause, it might actually be easier to just set `rent_stab_info` to `None` outside of the clause and set the session key to `rent_stab_info` after the logic is over, since there are a number of ways that `rent_stab_info` could end up being `None` (either because geosearch failed or because NYCDB isn't configured).

We should also add tests for this mutation to make sure the rent stab info is properly updated in either case!